cicd/forgeops-tests/config/6.5.0-manifest.txt

```diff
@@ -21,10 +21,10 @@ commons_version = 24.0.1
 amster_jvm = 1.8.0_151
 
 [openidm]
-version = 7.0.0-M1
-revision = 9c368d2
-date = 20190123140439
+version = 7.0.0-M2
+revision = 8a62b3d
+date = 20190213220443
 jdk_version = openjdk version "11.0.1" 2018-10-16
 jre_version = OpenJDK Runtime Environment (build 11.0.1+13-Debian-2bpo91)
 jdk_full_version = OpenJDK 64-Bit Server VM (build 11.0.1+13-Debian-2bpo91, mixed mode)
-commons_version = 25.0.0-alpha-14
\ No newline at end of file
+commons_version = 25.0.0-alpha-21
```

Comment:
this is really a 7.0 manifest, right?

Reply:
This file will be going away in the future, as we decided the best we can do right now is just to report on the various items and not attempt to validate against some "authoritative source" that doesn't exists.